py/pysparkling/ml/params.py

```diff
@@ -166,10 +165,7 @@ def setParallelizeCrossValidation(self, value):
         return self._set(parallelizeCrossValidation=value)
 
     def setDistribution(self, value):
-        assert_is_type(value, None, Enum("AUTO", "bernoulli", "quasibinomial", "modified_huber", "multinomial", "ordinal", "gaussian", "poisson", "gamma", "tweedie", "huber", "laplace", "quantile"))
```

Comment:
I wanted to reuse method defined on the Scala as defining the fields from enum manually is error prone. Plus created tests for the method validating enum values on Scala side.